CommonTools/ParticleFlow/python/pfCHS_cff.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+from CommonTools.ParticleFlow.pfNoPileUpJME_cff import adapt, pfPileUpJME
+from CommonTools.RecoAlgos.sortedPackedPrimaryVertices_cfi import sortedPackedPrimaryVertices
+
+packedPrimaryVertexAssociationJME = sortedPackedPrimaryVertices.clone(
+  produceSortedVertices = False,
+  producePileUpCollection  = False,
+  produceNoPileUpCollection = False
+)
+adapt(packedPrimaryVertexAssociationJME)
+
+pfCHS = cms.EDProducer("PFnoPileUpPacked",
```

Comment:
```suggestion
from CommonTools.ParticleFlow.pfNoPileUpPacked_cfi import pfNoPileUpPacked as _pfNoPileUpPacked 
pfCHS = _pfNoPileUpPacked.clone(
    vertexAssociationQuality=pfPileUpJME.vertexAssociationQuality
)
```
it's preferrable to reuse the fillDescription defaults when possible